zeppelin-client-examples/src/main/java/org/apache/zeppelin/client/examples/ZeppelinClientExample.java

```diff
@@ -41,6 +43,13 @@ public static void main(String[] args) throws Exception {
       noteId = zClient.createNote(notePath);
       System.out.println("Created note: " + noteId);
 
+      String newNotePath = notePath + "_rename";
+      zClient.renameNote(noteId, newNotePath);
+
+      NoteResult renamedNoteResult = zClient.queryNoteResult(noteId);
+      assert Objects.equals(newNotePath, renamedNoteResult.getNotePath());
```

Comment:
Not this assert, should be JUnit's assertEquals method

Reply:
got it, how about now?

Reply:
@zjffdu You are sure that you want to use the assert function of JUnit outside of a test class?